fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadManager.java

```diff
@@ -483,7 +484,7 @@ public RoutineLoadJob getJob(String dbFullName, String jobName) throws MetaNotFo
       if includeHistory is false, filter not running job in result
       else return all of result
      */
-    public List<RoutineLoadJob> getJob(String dbFullName, String jobName, boolean includeHistory)
+    public List<RoutineLoadJob> getJob(String dbFullName, String jobName, boolean includeHistory, PatternMatcher matcher)
             throws MetaNotFoundException {
```

Comment:
Please add a check " the 'jobName' and 'pattern' cannot exist at the same time" ~

Reply:
done